recipes/unidist/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This file can be deleted, the license is already included in the tar ball.

Reply:
I would prefer to keep it as we mention about the license in meta.yml.

Reply:
No it can be removed because it's a duplicate. 

Conda-build will look at the path you specify in the meta.yaml and the copy the files from there. 

So the only time you need to manually include the license is if it's not in the tar ball and cannot be copied from there.

Reply:
Ok, I see. Removed

Reply:
👍 I’m out now but will have a look at the rest later.

Reply:
A quick question. Where the license is taken from in the tar ball? Also, the guide recommends putting a license in meta.yml https://github.com/conda-forge/staged-recipes/blob/a504af81c05491bf7b0b018b2fa1efe64767985c/recipes/example/meta.yaml#L52-L55

Reply:
It's just copied from the source using the path you specify.

If you download https://files.pythonhosted.org/packages/1c/70/b965ef73cc52b3ffff287d1b7a941789b125ec53cffca27fdd917e272fef/unidist-0.1.0.tar.gz you'll find the license in the root of the package. This is the file conda-build copies. 
Since not every package includes the license, you can also include it manually here but that's only if it's not included in the source.

Yes but that'a just bad wording. What that means is that the license file should be included by specifying it there. Where the license file is coming from is not related to that.

Reply:
Yes, I see the License file. By the source you mean GH repo, but not PyPI url under `source` section, right? If so, the exact line is the following [home: https://github.com/modin-project/unidist](https://github.com/conda-forge/staged-recipes/pull/17920/files#diff-ef5dcc579071503164398c73b78a2fb0bc8fe937df8cf9e02d5ba79768ebb30eR130), correct?

Reply:
Basically, the process works like this:

1. Download source with source being the `source` section in the meta.yaml
2. Get license file path
3. Copy license from there into the newly created conda package.
4. If no file is there check if it was manually added to the recipe and copy it from there
5. If not there either, exit with error.

Reply:
Got it, thanks!